build_node/build_node_builder.py

```diff
@@ -208,7 +219,15 @@ def __report_done_task(self, task, success=True, artifacts=None):
             'status': 'done' if success else 'failed',
             'artifacts': [artifact.dict() for artifact in artifacts]
         }
-        self.__call_master('build_done', **kwargs)
+        try:
+            self.__call_master('build_done', **kwargs)
+        except Exception:
+            self.__logger.error(
+                f"Can't done task from master:\n"
```

Comment:
Can't mark the task as done ?

Can't done task sounds...unclear

Reply:
Fixed